packages/core/src/app/common/error/ErrorModal.tsx

```diff
@@ -68,6 +68,9 @@ export default class ErrorModal extends PureComponent<ErrorModalProps> {
 
         return (
             <>
+                {(error && isCustomError(error) && error.data.shouldBeTranslatedAsHtml) &&
```

Comment:
Could you please put `error.data.shouldBeTranslatedAsHtml` into the `error` interface of `ErrorModalProps`?

Reply:
I think you can write a typeguard for this check. Also is `shouldBeTranslatedAsHtml` something that is added new?

Reply:
added

Reply:
> I think you can write a typeguard for this check. Also is `shouldBeTranslatedAsHtml` something that is added new?

I added this flag that let us know if we need to handle it as html